### PR TITLE
Fix resolution of SASS modules when linking the library via `tsconfig` path overrides

### PR DESCRIPTION
**Pull request by danielwiehl**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [guidelines](https://github.com/SchweizerischeBundesbahnen/scion-toolkit/blob/master/CONTRIBUTING.md)
- [ ] Tests for the changes have been added
- [ ] Docs have been added or updated

## PR Type
What kind of change does this PR introduce?

- [x] Fix
- [ ] Feature
- [ ] Documentation
- [ ] Refactoring (changes that neither fixes a bug nor adds a feature)
- [ ] Performance (changes that improve performance)
- [ ] Test (adding missing tests, refactoring tests; no production code change)
- [ ] Chore (other changes like formatting, updating the license, removal of deprecations, etc)
- [ ] Deps (changes related to updating dependencies)
- [ ] CI (changes to our CI configuration files and scripts)
- [ ] Revert (revert of a previous commit)
- [ ] Release (publish a new release)
- [ ] Other... Please describe:

## Issue

Issue Number: #

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/scion/components/_index.scss [hunk 1]
```diff
@@ -1,23 +1 @@
...
- *       "es2015": "./fesm2015/scion-components.mjs",
- *       "node": "./fesm2015/scion-components.mjs",
- *       "default": "./fesm2020/scion-components.mjs"
- *     }
- *   }
- * }
- * ```
- */
-@forward './components';
+@forward './viewport';
```
Comment (mofogasy) [inline projects/scion/components/_index.scss:1]: Add newline

**Review by mofogasy — DISMISSED** [review]
LGTM

**Review by mofogasy — APPROVED** [review]
LGTM

File: projects/scion/components/_index.scss (hunk 1 above)
Comment (danielwiehl) [reply projects/scion/components/_index.scss]: done

**Pull request merged**